pymc3/sampling.py

```diff
@@ -55,6 +92,10 @@ def sample(draws, step, start=None, trace=None, chain=0, njobs=1, tune=None,
     -------
     MultiTrace object with access to sampling values
     """
+    model = modelcontext(model)
+    
+    step = assign_step_methods(model, step)
```

Comment:
OK, check out bdc9256. I have added a `Constant` step method that will simply keep its variables at its current value. Has the default competence of 0, so it will never be selected automatically.